### ISSUE TITLE
VPAT — Header Hierarchy Warnings In Discussion Forum

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#3559]
Related to issue #2309, PR #3557 changes breadcrumbs from `<h2>` to `<div>` to correct header hierarchy warnings noted by WAVE.  This works for most page views, but the discussion forum needs additional attention.

Please consider:
- Do not use header tags to style text.  Use CSS to style text.
- Header tags should follow a proper outline, with the top most tag always being `<h1>` 
- All views must have a top most `<h1>`.

Screenshot (note areas stroked in black):
![Header_Hierarchy](https://user-images.githubusercontent.com/12703624/55916936-c42ff500-5bbb-11e9-87dc-8026dabecd7f.png)

The screenshot does show additional VPAT warnings and errors that should be addressed.  This issue is intended to draw attention to the header hierarchy warnings as was noted in PR #3557.

**Comment by xprilion** [comment]
@andrewaikens87 @bmcutler I checked and compared the forum page to the other pages, which do not have any such errors and conclude that it will be tough to have an H1 element on the forum page. Kindly provide suggestions for including an H1 element.

**Comment by bmcutler** [comment]
Hi @xprilion,

We talked about this issue briefly in our group meeting today, and we agreed and thought it would be best to visually leave the discussion forum page layout as is, but add an invisible/hidden <h1>Discussion Forum</h1> somewhere on the page (size zero or positioned off screen) for screen reader context.  We didn't want to sacrifice any real estate on this complex and content-heavy page to visibly write that text a second time (it is already in the breadcrumb bar).

@pbailie, Do you think this is an ok plan for this page?

Note:  @mktung will be tackling the re-styling of the mobile version of this page, so the underlying code might need some refactoring to help that process.

**Comment by pbailie** [comment]
> @pbailie, Do you think this is an ok plan for this page?

Personally, I don't like it as it seems more of a hack to "trick" WAVE when there are actual standards to follow.  And while a useful tool, WAVE isn't the final word in accessible design.  But I do see how it is a significant challenge to reorganize the discussion forum.  It is very compact and doesn't easily reorganization.

When it comes to a campus accessibility review, my opinion won't be the only one that matters.  So, I'd like to get additional feedback from @mcglow2.  Unfortunately, @mcglow2 is out all week.  So is this something that needs more immediate feedback?  Or can it wait until @mcglow2 returns?

**Comment by bmcutler** [comment]
For now, let's add a hidden h1 element to the forum page to satisfy wave.  This cheat is ok for now since the page is sufficiently visually labeled with the breadcrumb.

Next time we do any significant refactor of the page styling/breadcrumb code, we will likely move the h1 to the breadcrumb and more elegantly and consistently satisfy this requirement.

**Comment by xprilion** [comment]
Fixing this in #3905 for sake of not creating another issue to remove inline CSS in this 1 line :p

**Comment by bmcutler** [comment]
Closed in PR #3905
